lib/java/com/google/android/material/datepicker/MaterialCalendar.java

```diff
@@ -298,6 +298,19 @@ CalendarConstraints getCalendarConstraints() {
   void setCurrentMonth(Month moveTo) {
     MonthsPagerAdapter adapter = (MonthsPagerAdapter) recyclerView.getAdapter();
     int moveToPosition = adapter.getPosition(moveTo);
+
+    // If target position is outside of the constraint bounds, adjust target to outer bounds
+    if (calendarConstraints != null) {
+      if (moveToPosition > calendarConstraints.getMonthSpan()) {
```

Comment:
Probably need change to:
```
moveToPosition >= calendarConstraints.getMonthSpan()
```

Reply:
and below
```
int offset = moveToPosition - calendarConstraints.getMonthSpan() + 1;
```

Reply:
@klblk I do not believe your suggestions are correct. `moveToPosition` is the position within the recyclerView. If `movePosition == calendarConstraints.getMonthSpan()` this indicates we will just be scrolling to the end of it and there is no need to offset the target position. This holds true for the minimum scenario as well and was verified in my personal testing. Is there a counter-case that you encountered that makes you think my solution is "off-by-one" ?

Reply:
@zkovar I debugged option "one year forward". 
February 2020 -> position = 0
February 2021 -> position = 12
calendarConstraints.getMonthSpan() = 13

if I select March 2020 and next switch year to 2021 -> movePosition = 13 and `moveToPosition > calendarConstraints.getMonthSpan()` false

but position 13 -> March 2021 and not in the range